frontend/scripts/actions/app.actions.js

```diff
@@ -200,3 +206,17 @@ export function loadSearchResults(searchTerm) {
       });
   };
 }
+
+export function selectYears(years) {
+  return (dispatch, getState) => {
+    const { tool } = getState();
+    dispatch({
+      type: SELECT_YEARS,
+      years
+    });
+    if (tool) {
```

Comment:
Change this check for `location.type === tool` there’s no need for loading nodes and links out of the tool page. And once the user visits and exits the tool page its reducer will be defined.